crates/sui-types/src/execution_mode.rs

```diff
@@ -147,7 +147,7 @@ impl ExecutionMode for System {
     }
 
     fn allow_arbitrary_values() -> bool {
-        false
+        true
```

Comment:
Could you explain (and add comment) why we need this?

Reply:
I'm a bit conflicted on this point.  On the one hand, `allow_arbitrary_values` is quite a heavy hammer to wield for this, and you could get away with not needing it by flattening out `vector<ActiveJwk>`, a number of ways:

- A different vector for each field.
- A vector to pack all the string fields and then a separate vector for the epoch.
- BCS-encoded in a single `vector<u8>` (and then using `sui::bcs` to parse out the individual structs).

But on the other hand, that is understandably a pain.

Reply:
@amnn My inclination right now is to leave this as is, unless you see a strong argument against not allowing system txns to do this. When I spoke to @tnowacki about this earlier his take was "well, we disabled it earlier because we didn't need it, but there's no big reason not to allow it"

Reply:
My main argument is that it means we need to do more due diligence around inputs to system transactions, because we could e.g. construct a coin and put it in there, breaking supply invariants.  We have checks and balances to catch supply invariants in particular, but generally it weakens the principle that all Move value constructors are protected within their modules.  To date we've only weakened that for dev-inspect which we don't persist results from.  Allowing arbitrary values in a transaction whose effects we do commit to the store felt like a different kettle of fish.

But, I may also be underestimating how special system transactions are already -- maybe this extra level of care we need to take is not significant relative to the other ways we need to treat system transactions specially.  If that's the case, feel free to ignore this comment.